docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc

```diff
@@ -137,6 +137,9 @@ using a ServiceLoader
 where the principal transformer is an aggregation of other principal
 transformers.
 
+|case-principal-transformer |A principal transformer definition where
+the principal transformer is adjusted to upper or lower case.
```

Comment:
This should be reworded slightly since it's the principal that is being adjusted (not the principal transformer)

Reply:
fixed